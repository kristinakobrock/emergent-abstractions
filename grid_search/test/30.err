  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 26.73it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 22.71it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 25.34it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 26.90it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 23.67it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 23.44it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 22.19it/s] 48%|############################################1                                               | 24/50 [00:01<00:01, 23.07it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 24.71it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 26.13it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 28.63it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 26.53it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 26.23it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 27.09it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 26.44it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 27.40it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.97it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.42it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 23.54it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 26.39it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.59it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.15it/s]
