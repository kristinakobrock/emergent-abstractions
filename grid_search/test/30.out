# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6951479315757751, "acc": 0.6008726358413696, "length": 2.115203857421875, "mode": "train", "epoch": 1}
{"loss": 0.6610478162765503, "acc": 0.6279297471046448, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6595187187194824, "acc": 0.6285142302513123, "length": 1.1090247631072998, "mode": "train", "epoch": 2}
{"loss": 0.6591206789016724, "acc": 0.6263671517372131, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6582427024841309, "acc": 0.6274306178092957, "length": 1.057111144065857, "mode": "train", "epoch": 3}
{"loss": 0.6636782288551331, "acc": 0.627734363079071, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6578023433685303, "acc": 0.6281049847602844, "length": 1.039352297782898, "mode": "train", "epoch": 4}
{"loss": 0.6607184410095215, "acc": 0.6300780773162842, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6570780873298645, "acc": 0.6282702684402466, "length": 1.0240910053253174, "mode": "train", "epoch": 5}
{"loss": 0.6635673642158508, "acc": 0.6302734017372131, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6584910154342651, "acc": 0.6274527311325073, "length": 1.016530156135559, "mode": "train", "epoch": 6}
{"loss": 0.6627732515335083, "acc": 0.6298827528953552, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6557415127754211, "acc": 0.6280965805053711, "length": 1.019408941268921, "mode": "train", "epoch": 7}
{"loss": 0.6594849228858948, "acc": 0.6273437142372131, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6469917893409729, "acc": 0.6381876468658447, "length": 1.4594510793685913, "mode": "train", "epoch": 8}
{"loss": 0.6368594169616699, "acc": 0.6548095345497131, "length": 1.9453125, "mode": "test", "epoch": 8}
{"loss": 0.6124157905578613, "acc": 0.6755843758583069, "length": 2.0731239318847656, "mode": "train", "epoch": 9}
{"loss": 0.6160138845443726, "acc": 0.6841309070587158, "length": 2.29296875, "mode": "test", "epoch": 9}
{"loss": 0.5834628939628601, "acc": 0.6974300146102905, "length": 2.329883337020874, "mode": "train", "epoch": 10}
{"loss": 0.5863103866577148, "acc": 0.702191174030304, "length": 2.87890625, "mode": "test", "epoch": 10}
{"loss": 0.56746506690979, "acc": 0.7109418511390686, "length": 2.8694093227386475, "mode": "train", "epoch": 11}
{"loss": 0.5748284459114075, "acc": 0.6981810927391052, "length": 2.98046875, "mode": "test", "epoch": 11}
{"loss": 0.5513284206390381, "acc": 0.7147730588912964, "length": 3.010110378265381, "mode": "train", "epoch": 12}
{"loss": 0.5667080283164978, "acc": 0.707489013671875, "length": 2.828125, "mode": "test", "epoch": 12}
{"loss": 0.5408048033714294, "acc": 0.7276201844215393, "length": 3.1463916301727295, "mode": "train", "epoch": 13}
{"loss": 0.5541181564331055, "acc": 0.7145934700965881, "length": 3.3203125, "mode": "test", "epoch": 13}
{"loss": 0.5332456231117249, "acc": 0.729542076587677, "length": 3.2482802867889404, "mode": "train", "epoch": 14}
{"loss": 0.553135097026825, "acc": 0.727813720703125, "length": 3.15625, "mode": "test", "epoch": 14}
{"loss": 0.521404504776001, "acc": 0.743280291557312, "length": 3.347994327545166, "mode": "train", "epoch": 15}
{"loss": 0.5394394993782043, "acc": 0.741381824016571, "length": 3.51953125, "mode": "test", "epoch": 15}
{"loss": 0.5152877569198608, "acc": 0.7453495264053345, "length": 3.4363479614257812, "mode": "train", "epoch": 16}
{"loss": 0.5536617636680603, "acc": 0.723419189453125, "length": 3.53125, "mode": "test", "epoch": 16}
{"loss": 0.4937131404876709, "acc": 0.7612073421478271, "length": 3.6071937084198, "mode": "train", "epoch": 17}
{"loss": 0.5149422883987427, "acc": 0.7518798112869263, "length": 3.70703125, "mode": "test", "epoch": 17}
{"loss": 0.46446794271469116, "acc": 0.7808699607849121, "length": 3.6723012924194336, "mode": "train", "epoch": 18}
{"loss": 0.5168664455413818, "acc": 0.760266125202179, "length": 3.8359375, "mode": "test", "epoch": 18}
{"loss": 0.4450379014015198, "acc": 0.7928550243377686, "length": 3.723459005355835, "mode": "train", "epoch": 19}
{"loss": 0.49624550342559814, "acc": 0.783465564250946, "length": 3.765625, "mode": "test", "epoch": 19}
{"loss": 0.41721099615097046, "acc": 0.8110767602920532, "length": 3.774843215942383, "mode": "train", "epoch": 20}
{"loss": 0.4664059579372406, "acc": 0.7884460091590881, "length": 3.90625, "mode": "test", "epoch": 20}
{"loss": 0.3925827741622925, "acc": 0.8247940540313721, "length": 3.8686840534210205, "mode": "train", "epoch": 21}
{"loss": 0.436635822057724, "acc": 0.8035582900047302, "length": 3.98828125, "mode": "test", "epoch": 21}
{"loss": 0.36660292744636536, "acc": 0.8394763469696045, "length": 3.906597137451172, "mode": "train", "epoch": 22}
{"loss": 0.4313396215438843, "acc": 0.8150207996368408, "length": 3.98046875, "mode": "test", "epoch": 22}
{"loss": 0.34608957171440125, "acc": 0.8531219959259033, "length": 3.931722402572632, "mode": "train", "epoch": 23}
{"loss": 0.4179617762565613, "acc": 0.8269529938697815, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.3288727104663849, "acc": 0.8585492968559265, "length": 3.982388734817505, "mode": "train", "epoch": 24}
{"loss": 0.4033704400062561, "acc": 0.8357421159744263, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.31117019057273865, "acc": 0.8715240359306335, "length": 3.9942686557769775, "mode": "train", "epoch": 25}
{"loss": 0.38507843017578125, "acc": 0.841796875, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.2897487282752991, "acc": 0.883455753326416, "length": 3.998077630996704, "mode": "train", "epoch": 26}
{"loss": 0.3690863251686096, "acc": 0.8544921278953552, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.2808854281902313, "acc": 0.8898400664329529, "length": 3.996509552001953, "mode": "train", "epoch": 27}
{"loss": 0.35167908668518066, "acc": 0.8621093034744263, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.27405399084091187, "acc": 0.8924594521522522, "length": 3.993182897567749, "mode": "train", "epoch": 28}
{"loss": 0.3486770689487457, "acc": 0.8685546517372131, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.25176456570625305, "acc": 0.9027841091156006, "length": 3.9992034435272217, "mode": "train", "epoch": 29}
{"loss": 0.35560813546180725, "acc": 0.8708983659744263, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.2475581020116806, "acc": 0.9038505554199219, "length": 3.9990296363830566, "mode": "train", "epoch": 30}
{"loss": 0.3308725357055664, "acc": 0.8787108659744263, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.23829199373722076, "acc": 0.909706175327301, "length": 3.9992516040802, "mode": "train", "epoch": 31}
{"loss": 0.3251532018184662, "acc": 0.8841796517372131, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.2339443564414978, "acc": 0.913192093372345, "length": 3.9983279705047607, "mode": "train", "epoch": 32}
{"loss": 0.33713462948799133, "acc": 0.8847655653953552, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.22555969655513763, "acc": 0.9165415167808533, "length": 3.9989795684814453, "mode": "train", "epoch": 33}
{"loss": 0.29189878702163696, "acc": 0.9019531607627869, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.21623879671096802, "acc": 0.9219202399253845, "length": 3.9996461868286133, "mode": "train", "epoch": 34}
{"loss": 0.33367347717285156, "acc": 0.8857421875, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.21733835339546204, "acc": 0.9208852648735046, "length": 3.9990739822387695, "mode": "train", "epoch": 35}
{"loss": 0.3135327696800232, "acc": 0.8976563215255737, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.20725630223751068, "acc": 0.9262117743492126, "length": 3.998887777328491, "mode": "train", "epoch": 36}
{"loss": 0.29822731018066406, "acc": 0.9054688215255737, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.2057785987854004, "acc": 0.9275723695755005, "length": 3.998896837234497, "mode": "train", "epoch": 37}
{"loss": 0.304780513048172, "acc": 0.8994141221046448, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.2060965895652771, "acc": 0.9274711608886719, "length": 3.9996871948242188, "mode": "train", "epoch": 38}
{"loss": 0.2890647053718567, "acc": 0.8994140625, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.19685395061969757, "acc": 0.9323588013648987, "length": 3.9999635219573975, "mode": "train", "epoch": 39}
{"loss": 0.31278419494628906, "acc": 0.8964844346046448, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.19401417672634125, "acc": 0.9309240579605103, "length": 3.9998815059661865, "mode": "train", "epoch": 40}
{"loss": 0.30036869645118713, "acc": 0.9068360328674316, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.2040105015039444, "acc": 0.9284262657165527, "length": 3.9999825954437256, "mode": "train", "epoch": 41}
{"loss": 0.3081813156604767, "acc": 0.9000000357627869, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.1984337419271469, "acc": 0.9294957518577576, "length": 3.999802350997925, "mode": "train", "epoch": 42}
{"loss": 0.3157367706298828, "acc": 0.9011718034744263, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.18842236697673798, "acc": 0.9336695671081543, "length": 3.999872922897339, "mode": "train", "epoch": 43}
{"loss": 0.3085026144981384, "acc": 0.9007812142372131, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.193109393119812, "acc": 0.931584894657135, "length": 3.9999823570251465, "mode": "train", "epoch": 44}
{"loss": 0.323642760515213, "acc": 0.9017577767372131, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.18703174591064453, "acc": 0.9350249767303467, "length": 3.9991843700408936, "mode": "train", "epoch": 45}
{"loss": 0.2906074523925781, "acc": 0.9076171517372131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.18056060373783112, "acc": 0.9380784630775452, "length": 3.999934434890747, "mode": "train", "epoch": 46}
{"loss": 0.2903050184249878, "acc": 0.9146484732627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.1811249852180481, "acc": 0.9386769533157349, "length": 3.9999964237213135, "mode": "train", "epoch": 47}
{"loss": 0.2944362163543701, "acc": 0.9124999642372131, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.18235594034194946, "acc": 0.9355774521827698, "length": 3.9999840259552, "mode": "train", "epoch": 48}
{"loss": 0.319372296333313, "acc": 0.9052734375, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.17640717327594757, "acc": 0.9389008283615112, "length": 3.999413251876831, "mode": "train", "epoch": 49}
{"loss": 0.3000488579273224, "acc": 0.9080077409744263, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.18322624266147614, "acc": 0.9351575970649719, "length": 3.999912738800049, "mode": "train", "epoch": 50}
{"loss": 0.3459070324897766, "acc": 0.8925781846046448, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.19068573415279388, "acc": 0.9349198937416077, "length": 3.9999074935913086, "mode": "train", "epoch": 51}
{"loss": 0.331393837928772, "acc": 0.8978515267372131, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.1911662220954895, "acc": 0.9350311756134033, "length": 3.999664545059204, "mode": "train", "epoch": 52}
{"loss": 0.32858920097351074, "acc": 0.9011720418930054, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.16870883107185364, "acc": 0.9438013434410095, "length": 3.9999680519104004, "mode": "train", "epoch": 53}
{"loss": 0.3047458529472351, "acc": 0.9128905534744263, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.17273610830307007, "acc": 0.9434990286827087, "length": 3.9998292922973633, "mode": "train", "epoch": 54}
{"loss": 0.30120396614074707, "acc": 0.9074220061302185, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.17724686861038208, "acc": 0.9386724233627319, "length": 3.9999074935913086, "mode": "train", "epoch": 55}
{"loss": 0.32395899295806885, "acc": 0.9031249284744263, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.1871495544910431, "acc": 0.9342668056488037, "length": 3.9999938011169434, "mode": "train", "epoch": 56}
{"loss": 0.3361356556415558, "acc": 0.8994141221046448, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.15915021300315857, "acc": 0.9476161003112793, "length": 3.999997138977051, "mode": "train", "epoch": 57}
{"loss": 0.32259824872016907, "acc": 0.9021483659744263, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.16467705368995667, "acc": 0.9436230659484863, "length": 3.9999892711639404, "mode": "train", "epoch": 58}
{"loss": 0.3152410686016083, "acc": 0.9089843034744263, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.16708791255950928, "acc": 0.9440996646881104, "length": 3.9999773502349854, "mode": "train", "epoch": 59}
{"loss": 0.31336623430252075, "acc": 0.9105468392372131, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.16204707324504852, "acc": 0.9452916979789734, "length": 3.999988079071045, "mode": "train", "epoch": 60}
{"loss": 0.3194364011287689, "acc": 0.9064453840255737, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.8425000309944153
